android/trifle/src/main/java/app/cash/trifle/KeyHandle.kt

```diff
@@ -35,6 +37,21 @@ class KeyHandle internal constructor(private val alias: String) {
     }
   }
 
+  internal fun getKeyPair(): KeyPair? {
```

Comment:
Instead of re-computing the keypair, should we make this a lazy var?

Reply:
yes, done